src/styles.scss

```diff
@@ -2,4 +2,5 @@
 body {
   background: #000;
   color: #fff;
+  font-family: 'Roboto Slab', serif;
```

Comment:
I think this is better with `font-family: 'Roboto Slab', sans-serif;` against google web fonts' recommendation. Find out the difference between sans serif and serif. I think sans serif is better for content and reading.